flink-end-to-end-tests/flink-glue-schema-registry-test/pom.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>flink-end-to-end-tests</artifactId>
+		<groupId>org.apache.flink</groupId>
+		<version>1.13-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>flink-glue-schema-registry-test</artifactId>
+	<name>Flink : E2E Tests : AWS Glue Schema Registry</name>
+	<packaging>jar</packaging>
+
+	<properties>
+		<httpclient.version>4.5.3</httpclient.version>
+		<httpcore.version>4.4.6</httpcore.version>
+		<aws.sdk.version>1.11.754</aws.sdk.version>
+		<aws.sdkv2.version>2.15.32</aws.sdkv2.version>
+	</properties>
+
+	<!-- ============================= -->
+	<!-- DEPENDENCY MANAGEMENT -->
+	<!-- ============================= -->
+	<dependencyManagement>
```

Comment:
Why is there a dependency management block here? Unless I am mistaken `dependencyManagement` is used to setup dependencies for child modules, that include this pom as their parent. How is this being used in this context?

Reply:
dependency management sections can be useful even in the absence of child modules because you can modify versions of transitive dependencies without adding an explicit dependency, which would change the dependency and obfuscate where the dependency actually comes from.

Reply:
I did not know that, thanks! That sounds like a much better approach than adding explicit dependency 👍